src/main/resources/MessagingDescriptor.json

```diff
@@ -5,8 +5,14 @@
       "description": "Patron fee/fine balance was changed",
       "eventTTL": 1,
       "signed": false
+    },
+    {
+      "eventType": "FF_ACCOUNT_WITH_LOAN_CLOSED",
+      "description": "An account with a loan associated is closed",
+      "eventTTL": 30,
```

Comment:
Currently `eventTTL` for all events is hardcoded to `1`, see `PubSubService#createEvent()`. What does it do anyway? There is no mention of it in mod-pubsub documentation...

Reply:
As far as I know from the Folijet team this flag is not implemented yet. It will control how long the message will 'live' in the queue. 

I'll probably revert it to 1 as well.

Reply:
Do you happen to also know which time unit it is supposed to represent? (seconds, minutes, etc.)

Reply:
it is time to live in minutes.